sql_grader/tests/test_display.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+"""
+Test basic XBlock display function
+"""
+from __future__ import absolute_import
```

Comment:
You shouldn't need this import now that we've dropped Python 2.7 support.

Reply:
Done.